packages/core/src/models/Library.ts

```diff
@@ -33,19 +25,17 @@ export const ComponentInfo = {
   },
 };
 
-export interface ComponentMetadata {
-  id: string;
-  name?: string;
-  description?: string;
-}
-
 export interface Component {
   from: string;
   href: string;
```

Comment:
how about deleting that one ? I'm not sure it's used anymore.

Reply:
Done